WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift

```diff
@@ -579,3 +583,12 @@ private extension String {
         static let title = NSLocalizedString("Loading Plugin...", comment: "Text displayed while loading an specific plugin")
     }
 }
+
+extension PluginViewModel {
+    func networkStatusDidChange(active: Bool) {
+        if active, case .error = state {
```

Comment:
It would be good to also cover the case when we have partial information of the plugin (loaded from the plugin list).
I would do something like this:
```swift
 if active {
    switch state {
        case .error:
            store.processQueries()
             state = .loading
         case .directoryEntry(let entry):
             store.processQueries()
             state = .directoryEntry(entry)
         default:
                break
        }
}
```

Note that calling `processQueries()` first, and then setting the `state` solves the error empty view issue.
If there's a better way of solving that would be cool!